src/components/ProjectConfigurationModal/ProjectConfigurationModal.js

```diff
@@ -193,14 +193,20 @@ const DisabledText = styled.div`
   color: ${COLORS.gray[500]};
 `;
 
-const mapStateToProps = (state, ownProps) => {
+const mapStateToProps = state => {
   const project = getSelectedProject(state);
   const projectId = project && project.id;
 
+  // This component is rendered even when no project is selected.
+  // In that case, this prop isn't applicable.
+  const dependenciesChangingForProject = projectId
```

Comment:
Nit, I think `isQueueEmpty` should return undefined if `projectId` is null. So I think it's better to add a check to `isQueueEmpty`. e.g. `return projectId && !getNextActionForProjectId(state, props)`

Reply:
Yeah, good call :+1: